src/main/target/common_fc_pre.h

```diff
@@ -26,7 +26,7 @@
 //#pragma GCC diagnostic warning "-Wpadded"
 
 //#define SCHEDULER_DEBUG // define this to use scheduler debug[] values. Undefined by default for performance reasons
-#define DEBUG_MODE DEBUG_NONE // change this to change initial debug mode
+#define DEBUG_MODE DEBUG_GYRO_UNFILTERED // change this to change initial debug mode
```

Comment:
What is the reasoning behind making this the default? In terms of possible logging rate, wouldn't it be better to have debug logging off by default, and let the user enable it if they are specifically interested in getting debug info for something?

Reply:
Default is only intended for RC4, I should have stated that. The plan is to turn it off for 3.2 release.